src/converter.ts

```diff
@@ -257,24 +257,14 @@ export class Converter {
 
   private static async runBrowser() {
     if (!Converter.browser) {
-      // Fix the rendered position of elements in <foreignObject>
-      // See: https://bugs.chromium.org/p/chromium/issues/detail?id=467484
-      const args = ['--enable-blink-gen-property-trees']
```

Comment:
Puppeteer v1.13.0 no longer works BGPT feature.
https://github.com/GoogleChrome/puppeteer/commit/ef2251d7a722bcd6d183f7876673224ac58f2244

It seems to have no critical problems while rendering PDF and images.